### ISSUE TITLE
Add a way to check the version

### ISSUE THREAD

**Issue by Chouby** [soflyy/wp-all-import-rapid-addon#8]
Hi,

There could be cases were a user has activated 2 addons, both bundled with the RapidAddon class. There is of course a `class_exists('RapidAddon')` to avoid the class to be redefined.

But boths addon could use the class at a different version. That means that one addon may expect functionalities which are not available and thus this could cause conflicts.

Also, without version number, it's hard to follow updates of the class. Do you plan to version the class? Or maybe the best would even to bundle it with WP All Import?

**Comment by cmdmcs** [comment]
Thanks @Chouby! I added version number and we will keep it updated. As for bundle class to WP All Import, probably we'll do this in near future.
